modules/store/src/provide_store.ts

```diff
@@ -67,6 +67,19 @@ import {
   _provideForRootGuard,
 } from './store_config';
 
+export function provideState<T, V extends Action = Action>(
+  featureName: string,
+  reducers: ActionReducerMap<T, V> | InjectionToken<ActionReducerMap<T, V>>,
+  config?: StoreConfig<T, V> | InjectionToken<StoreConfig<T, V>>
+): EnvironmentProviders;
+export function provideState<T, V extends Action = Action>(
+  featureName: string,
+  reducer: ActionReducer<T, V> | InjectionToken<ActionReducer<T, V>>,
+  config?: StoreConfig<T, V> | InjectionToken<StoreConfig<T, V>>
+): EnvironmentProviders;
+export function provideState<T, V extends Action = Action>(
+  slice: FeatureSlice<T, V>
+): EnvironmentProviders;
 /**
  * Provides additional slices of state in the Store.
```

Comment:
JSDoc comments are moved above the implementation, to be visible in the generated API docs.